blueprints/secops/sops/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: sops
+matrix:
+  versions:
+    - 3.8.0
+    - 3.7.3
+    - 3.7.2
+  architectures:
+    - amd64
+    - arm64
+homepage: https://age-encryption.org/
+summary: Simple and flexible tool for managing secrets
+description: |-
+  sops is an editor of encrypted files that supports YAML, JSON, ENV, INI and
```

Comment:
Could you also reformat the description here, put everything on the same line and run ops2deb format.

Reply:
thx fixed